Firmware/ultralcd.cpp

```diff
@@ -1605,9 +1605,7 @@ static void lcd_support_menu()
 
   MENU_ITEM_BACK_P(PSTR("Firmware:"));
   MENU_ITEM_BACK_P(PSTR(" " FW_VERSION_FULL));
-#if (FW_DEV_VERSION != FW_VERSION_GOLD) && (FW_DEV_VERSION != FW_VERSION_RC)
-  MENU_ITEM_BACK_P(PSTR(" repo " FW_REPOSITORY));
-#endif
+//  MENU_ITEM_BACK_P(PSTR(" repo " FW_REPOSITORY));
```

Comment:
why was this disabled? CMake attempts to set it....

Reply:
Fixed